### ISSUE TITLE
[T8][Accessibility][Functionality]Unable to select options from Display filter dropdown in Social Histories Tab

### ISSUE THREAD

**Issue by nehabug** [Mohanme13/Feb-19-Bug-bash#357]
Unable to select options from Display filter dropdown in Social Histories Tab through keyboard.
